vmdb/app/assets/javascripts/cfme_application.js

```diff
@@ -319,8 +319,8 @@ function toggleConvertButtonToLink(button, url, toggle) {
 // parms: button_div=<id of div with buttons to update>, override=<forced state>
 function miqUpdateAllCheckboxes(button_div,override) {
   miqSparkle(true);
-  if ($('masterToggle')) {
-    var state = $('masterToggle').checked;
+  if ($j('#masterToggle')) {
```

Comment:
@dclarizio you are right i will have to fix that, i can check $j('#masterToggle').length > 0 to make sure element id exists in DOM.